frontend/src/pull-card/index.tsx

```diff
@@ -38,9 +39,10 @@ const SigsAndFlags = chakra(Flex, {
    }
 });
 
-export function PullCard({pull}: {pull: Pull}) {
+export const PullCard = memo(
+function PullCard({pull, show}: {pull: Pull, show: boolean}) {
    return (
-      <Card position="relative">
+      <Card position="relative" display={show ? "" : "none"}>
```

Comment:
Super nit-picky, but I feel like this is usually a little more intuitive than an empty string? Looks like Chakra was typed to support it.

```suggestion
      <Card position="relative" display={show ? undefined : "none"}>
```

Reply:
Thanks! Good idea.